### PR TITLE
add change PUK functionality

### PR DESCRIPTION
**Pull request by bitgamma**
Add Change PUK functionality in Settings.

Addresses half of #10031. Attached screenshots for localization

status: ready

### CODE REVIEW

**Review by rasom — APPROVED** [review]

File: src/status_im/keycard/core.cljs [hunk 1]
```diff
@@ -307,6 +310,13 @@
                                            :original     []
                                            :confirmation []}))
 
+(defn- puk-enter-error [fx error-label]
+  (update-in fx [:db :keycard :pin] merge {:status       :error
+                                           :error-label  error-label
+                                           :enter-step   :puk-original
+                                           :puk-original     []
+                                           :puk-confirmation []}))
+
```
Comment (shivekkhurana) [inline src/status_im/keycard/core.cljs:319]: Map not formatted.

File: src/status_im/keycard/change_pin.cljs [hunk 2]
```diff
@@ -71,6 +84,26 @@
...
+    (fn [{:keys [db] :as cofx}]
+      (let [puk (common/vector->string
+                 (get-in db [:keycard :pin :puk-original]))
+            pin (common/vector->string
+                 (get-in db [:keycard :pin :current]))]
+        (fx/merge
+         cofx
+         {:db (assoc-in db [:keycard :pin :status] :verifying)
+          :keycard/change-puk
+          {:puk     puk
```
Comment (shivekkhurana) [inline src/status_im/keycard/change_pin.cljs]: Map not formatted.

**Review by shivekkhurana — APPROVED** [review]

File: src/status_im/keycard/change_pin.cljs (hunk 2 above)
Comment (bitgamma) [reply src/status_im/keycard/change_pin.cljs]: Fixed, thanks!

**Comment by qoqobolo** [comment]
No regression issues were found, e2e failures are not PR related. It's ready to be merged.
@bitgamma thank you for the great work!

**Pull request merged**